tests/test_other.py

```diff
@@ -10327,6 +10327,27 @@ def test_split_module(self):
     self.assertNotIn('profile', result)
     self.assertIn('Hello! answer: 42', result)
 
+  def test_loadSplitModule(self):
```

Comment:
How about `test_load_split_module` to be consistent with the test naming convention? Also, @sbc100, do you think it would be better to deduplicate the logic here with the previous test? They're identical except for the post-js file and the extra assertion in this test.

Reply:
I agree there is two much duplication here between these different split module tests.  

Perhaps we can just modify the original test to have two different modes?   Using `@parameterized`.   This variant can inject a second `--post-js=` which just adds the `Module["loadSplitModule"] = function ...` and then checks for the output.